rollup.config.js

```diff
@@ -28,16 +28,7 @@ export default {
   plugins: [
     clear({ targets: ['dist'] }),
     replace({
-      patterns: [
-        {
-          match: /lib(\/|\\)main/,
-          test: /(\/\/ #NO_GLOBAL)(.|\r|\n)*\1/,
-          /* export default function canvasDatagrid (args) {
-				  return new Grid(args);
-				}; */
-          replace: ``,
-        },
-      ],
+      'window.EXCLUDE_GLOBAL': true,
```

Comment:
I think `true` needs to be quoted, no? My linter complains:

<img width="692" alt="rollup_config_js_—_canvas-josh" src="https://user-images.githubusercontent.com/950979/100257189-eff51f80-2f45-11eb-83b7-2d6dd7f4619c.png">

If you fix this, and add a comment to `lib/main.js` explaining what the use of `EXCLUDE_GLOBAL` is for (i.e. exporting as an ES module, instead of setting on `window`), I'll merge this.

Reply:
The documentation isn't clear, it says any primitives should be valid, but it looks like they really want everything to be `JSON.stringify()`ed ideally. I'll add it that way.
Just confirmed that it does get tree-shaken with it being stringified.

Reply:
Ah, I guess that's maybe the type definition, then. This is better, but could you change `JSON.stringify(true)` to `'true'` (i.e., just quoting it as a string)?

The former seems needlessly complex in the event of a simple plain value like `true` :-)